yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1602,6 +1603,14 @@ func postProcessingOfAssessmentReport() {
 func generateAssessmentReportJson(reportDir string) error {
 	jsonReportFilePath := filepath.Join(reportDir, fmt.Sprintf("%s%s", ASSESSMENT_FILE_NAME, JSON_EXTENSION))
 	log.Infof("writing assessment report to file: %s", jsonReportFilePath)
+
+	var err error
+	assessmentReport.MigrationComplexityExplanation, err = buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		utils.PrintAndLog("ERROR: unable to build migration complexity explanation for json report: %v", err)
```

Comment:
why `utils.PrintAndLog`, use return `fmt.Errorf(..)` ?

Reply:
I was thinking of showing something on console also for users to know that explanation is not there for xyz reason

Reply:
The top level functions which is calling the generateAssessmentReportJson must be printing the error on console for users so it will come eventually right?

Reply:
will this not look good? any suggestions?

Reply:
> The top level functions which is calling the generateAssessmentReportJson must be printing the error on console for users so it will come eventually right?

No, what i did was to not error out the cmd if building explanation fails...

Reply:
Oh, I see. Ideally, we should return the error,and mostly the errors occur while `buildMigrationComplexityExplanation ` is generating HTML string, etc.
For the case, we should fail the command for this new addition, we can use env var approach and user faces any error we can always ask them to disable it.

Reply:
hmm makes sense, this is the standard code - creating template and executing it.
Should not be error here ideally.

Reply:
done